.changeset/spotty-shirts-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@animareflection/ui": patch
+---
+
+Update `Accordion` component
```

Comment:
We can do this by updating the changelog in release PRs later on, but best to describe the changes made (e.g. in this case, "Added open & close transition styles to `Accordion` component" or similar)

Reply:
Understood. Moving forward, I'll ensure to provide descriptive updates in the changelog within release PRs. Thank you for pointing it out.